cmd/parca-agent/main.go

```diff
@@ -320,7 +320,7 @@ func run(logger log.Logger, reg *prometheus.Registry, flags flags) error {
 		),
 	}
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		if r.URL.Path == "/favicon.ico" {
+		if r.URL.Path == "/healthy" || r.URL.Path == "/ready" || r.URL.Path == "/favicon.ico" {
```

Comment:
nit: What do you think about`/status`? I think it is a more common health check path, no?

Reply:
`ready` and `healty` is a common naming for the Kubernetes-era, Prometheus, Thanos, and Kubernetes components use it for example. Later, we can imagine more in-depth checks under `/ready` and keep things light for continuous `/healthy` checks

Reply:
Sounds good! Guess I am not up to date on the Cloud Native ™️  lingo!